src/Arcus.Template.WebApi/Dockerfile

```
@@ -0,0 +1,23 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:2.2-stretch-slim AS base
```

We should not be using strecht but alpine instead as it's smaller and more secure
```suggestion
FROM mcr.microsoft.com/dotnet/core/aspnet:2.2.7-alpine3.9 AS base
```

Let's pin to a specific version while we're at it.